src/db/sql/events-actions/queries.ts

```diff
@@ -106,8 +106,23 @@ function emittedZkAppCommandsCTE(db_client: postgres.Sql) {
   return db_client`
   emitted_zkapp_commands AS (
     SELECT
-      blocks_accessed.*,
+      blocks_accessed.requesting_zkapp_account_identifier_id,
```

Comment:
Expanding `*` queries is optional, but I think it will be more clear for the next person who has to dig here.  I struggled to make sense of these queries at first.